src/vs/base/browser/ui/inputbox/inputBox.ts

```diff
@@ -392,6 +406,7 @@ export class InputBox extends Widget {
 				const styles = this.stylesForType(this.message.type);
 				spanElement.style.backgroundColor = styles.background ? styles.background.toString() : null;
 				spanElement.style.border = styles.border ? `1px solid ${styles.border}` : null;
+				spanElement.style.wordWrap = 'break-word';
```

Comment:
I would prefer if we added this in the inputBox.css here https://github.com/Microsoft/vscode/blob/isidorn/searchPanel/src/vs/base/browser/ui/inputbox/inputBox.css#L88